internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -114,6 +126,15 @@ func (h *HighPerformanceHooks) PreStop(ctx context.Context, c *oci.Container, s
 		}
 	}
 	// no need to reverse the cgroup CPU CFS quota setting as the pod cgroup will be deleted anyway
+	// Allow the container CPUs to be used by the net devices soft IRQs
```

Comment:
I would probably add an empty line here, this way the two unrelated comments can be confusing

Reply:
ok, thanks